docs/assets/bindings-delete-flow.drawio

```diff
@@ -0,0 +1,101 @@
+<mxfile host="Electron" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/24.7.17 Chrome/128.0.6613.36 Electron/32.0.1 Safari/537.36" version="24.7.17">
+  <diagram name="Page-1" id="DYFdJtEx81X-teSN2fxA">
+    <mxGraphModel dx="1506" dy="630" grid="0" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="0" pageScale="1" pageWidth="850" pageHeight="1100" background="#ffffff" math="0" shadow="0">
+      <root>
+        <mxCell id="0" />
```

Comment:
We `build kubeconfig with generated token` and then use `generated kubeconfig`, `generated Kubeconfig`.
This seems inconsistent.
Could we `get/fetch kubeconfig with token`?